tox.ini

```diff
@@ -1,15 +1,41 @@
 [tox]
 skipsdist = true
-envlist = py27,py36
+envlist =
+    {py27,py36,pypy}-{dj11}
```

Comment:
want to add diff drf versions? remember something changed in serializers a while back so maybe testing multiple versions will help. thoughts?

Reply:
`3.8` and `3.9` should do